<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDialog</class>
 <widget class="QWidget" name="GameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicTab">
      <attribute name="title">
       <string>Normal Games</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="opponentGroup">
         <property name="title">
          <string>Opponent</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="oppHuman">
            <property name="text">
             <string>Human at the same computer</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="compLayout">
            <item>
             <widget class="QRadioButton" name="oppComp">
              <property name="text">
               <string>Computer engine:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KHistoryComboBox" name="programComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="historyItems">
               <stringlist/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ficsLayout">
            <item>
             <widget class="QRadioButton" name="oppFics">
              <property name="text">
               <string>Human at a chess server:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KHistoryComboBox" name="serverComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="historyItems">
               <stringlist/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="colorGroup">
         <property name="title">
          <string>Your Color</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="colorWhite">
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="colorBlack">
            <property name="text">
             <string>Black</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="colorRandom">
            <property name="text">
             <string>Choose randomly</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="timeGroupBasic">
         <property name="title">
          <string>Time Limit</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="numberOfMovesLabel">
            <property name="text">
             <string>Number of &amp;moves</string>
            </property>
            <property name="buddy">
             <cstring>numberOfMoves</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="startingTimeLabel">
            <property name="text">
             <string>Starting &amp;Time</string>
            </property>
            <property name="buddy">
             <cstring>startingTime</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="timeIncrementLabel">
            <property name="text">
             <string>Time &amp;Increment</string>
            </property>
            <property name="buddy">
             <cstring>timeIncrement</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KIntSpinBox" name="numberOfMoves"/>
          </item>
          <item row="1" column="1">
           <widget class="QTimeEdit" name="startingTime"/>
          </item>
          <item row="2" column="1">
           <widget class="QTimeEdit" name="timeIncrement"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="customTab">
      <attribute name="title">
       <string>Custom</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="whiteBox">
         <property name="title">
          <string>White</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="whiteHuman">
            <property name="text">
             <string>Human</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="compLayout_2">
            <item>
             <widget class="QRadioButton" name="whiteComp">
              <property name="text">
               <string>Computer engine:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KHistoryComboBox" name="whiteProgram">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="historyItems">
               <stringlist/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ficsLayout_2">
            <item>
             <widget class="QRadioButton" name="whiteFics">
              <property name="text">
               <string>Human at a chess server:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KHistoryComboBox" name="whiteServer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="historyItems">
               <stringlist/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="blackBox">
         <property name="title">
          <string>Black</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="blackHuman">
            <property name="text">
             <string>Human</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="compLayout_3">
            <item>
             <widget class="QRadioButton" name="blackComp">
              <property name="text">
               <string>Computer engine:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KHistoryComboBox" name="blackProgram">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="historyItems">
               <stringlist/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ficsLayout_3">
            <item>
             <widget class="QRadioButton" name="blackFics">
              <property name="text">
               <string>Human at a chess server:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KHistoryComboBox" name="blackServer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="historyItems">
               <stringlist/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="timeGroupBasic_2">
         <property name="title">
          <string>Time Limit</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="numberOfMovesLabel_2">
            <property name="text">
             <string>Number of &amp;moves</string>
            </property>
            <property name="buddy">
             <cstring>numberOfMoves_2</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="startingTimeLabel_2">
            <property name="text">
             <string>Starting &amp;Time</string>
            </property>
            <property name="buddy">
             <cstring>startingTime_2</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="timeIncrementLabel_2">
            <property name="text">
             <string>Time &amp;Increment</string>
            </property>
            <property name="buddy">
             <cstring>timeIncrement_2</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KIntSpinBox" name="numberOfMoves_2"/>
          </item>
          <item row="1" column="1">
           <widget class="QTimeEdit" name="startingTime_2"/>
          </item>
          <item row="2" column="1">
           <widget class="QTimeEdit" name="timeIncrement_2"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>numberOfMoves</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numberOfMoves_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startingTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>startingTime_2</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeIncrement</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeIncrement_2</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberOfMoves_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numberOfMoves</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startingTime_2</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>startingTime</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeIncrement_2</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeIncrement</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oppComp</sender>
   <signal>toggled(bool)</signal>
   <receiver>programComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oppFics</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oppHuman</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorGroup</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteComp</sender>
   <signal>toggled(bool)</signal>
   <receiver>whiteProgram</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteFics</sender>
   <signal>toggled(bool)</signal>
   <receiver>whiteServer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackComp</sender>
   <signal>toggled(bool)</signal>
   <receiver>blackProgram</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackFics</sender>
   <signal>toggled(bool)</signal>
   <receiver>blackServer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteFics</sender>
   <signal>toggled(bool)</signal>
   <receiver>blackFics</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackFics</sender>
   <signal>toggled(bool)</signal>
   <receiver>whiteFics</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sameTimeChanged(bool)</slot>
  <slot>timeEnabled(bool)</slot>
  <slot>hotseatModeToggled(bool)</slot>
 </slots>
</ui>
